updateOpenFinConfigForDeployment.js

```diff
@@ -0,0 +1,29 @@
+const semver = require('semver');
+const configBuilder = require('openfin-config-builder');
+
+if (process.argv.length <= 2) {
+    console.error(`Usage: node ${__filename} <deployName>`);
+    process.exit(1);
+}
+
+const deployName = process.argv[2];
+const deployedUrl = `http://rjmcneill.github.io/StockFlux/${deployName}/`;
```

Comment:
I'd move this after the `deployName` validation check and before the OpneFin config update, around line 15 (so it's declared and defined closer to where it's first used).